platform/core.multitabs/src/org/netbeans/core/multitabs/impl/ControlsToolbar.java

```diff
@@ -58,6 +58,7 @@ public void updateUI() {
     private void configure() {
         setFloatable( false );
         setFocusable( false );
+        setOpaque( false );
```

Comment:
Did you check to make sure this did not cause problems in other LAFs?

Reply:
Yes, tested with Window, Mac, Nimbus and Metal LaFs. `ControlsToolbar` is a package private class and only used in in `AbstractTabDisplayer`.